litellm/llms/cohere/completion.py

```diff
@@ -153,7 +153,12 @@ def completion(
     headers = validate_environment(api_key, headers=headers)
     completion_url = api_base
     model = model
-    prompt = " ".join(message["content"] for message in messages)
+    prompt = ""
+    for message in messages:
+        if isinstance(message["content"], str):
+            prompt = prompt + " " + message["content"]
+        else:
+            prompt = prompt + "\n" + " ".join(item.get("text") for item in message["content"])
```

Comment:
Due to change in structure
Previous:
```json
{
  "messages": [
    {
      "role": "system",
      "content": "What is automation testing ",
      "name": null,
      "function_call": null
    }
  ]
}
```
Current :
```json
{
  "messages": [
    {
      "role": "system",
      "content": [
        {
          "type": "text",
          "text": "What is automation testing "
        }
      ],
      "name": null,
      "function_call": null
    }
  ]
}
```

Reply:
We should be doing this in raga_utils

Reply:
removed